src/main/java/com/koichi/assignment8/mapper/StudentMapper.java

```diff
@@ -14,12 +17,22 @@ public interface StudentMapper {
     @Select("SELECT * FROM students")
     List<Student> findAllStudents();
 
-    @Select("SELECT *  from students WHERE school_year LIKE CONCAT(#{grade}, '%') ")
+    @Select("SELECT *  from students WHERE grade LIKE CONCAT(#{grade}, '%') ")
     List<Student> findByGrade(Integer grade);
 
     @Select("SELECT * FROM students WHERE name LIKE CONCAT(#{startsWith}, '%') ")
     List<Student> findByName(String startsWith);
 
     @Select("SELECT * FROM students WHERE birth_place LIKE CONCAT(#{birthplace}, '%') ")
     List<Student> findByBirthPlace(String birthPlace);
+
+
+    /**
+     * INSERT用のMapper
+     */
+
+    @Insert("INSERT INTO students (name,grade,birth_place) VALUES (#{name}, #{grade},#{birthPlace})")

```

Comment:
javadoc いれるの good
メソッドとの間に空行はいらないです 👍 
```suggestion
    /**
     * INSERT用のMapper
     */
    @Insert("INSERT INTO students (name,grade,birth_place) VALUES (#{name}, #{grade},#{birthPlace})")

```

Reply:
- javadocの修正
 - 7a9a842f6784866422c66b7913b1f38c2a81aaac

Reply:
👌